src/modules/job-list/match.c

```diff
@@ -743,6 +831,28 @@ struct match_ctx *match_ctx_create (flux_t *h)
         goto error;
     }
 
+    if (flux_get_size (mctx->h, &mctx->max_hostlist) < 0)
+        goto error;
```

Comment:
`errp->text` not set here.

Reply:
i think you mean I forgot a flux_log in this case, but correct!!

Reply:
Oh, heh. Yeah I must have been blindly checking `goto error` without `errprintf()` :facepalm: